src/unum/devtelemetry/dt_collector.c

```diff
@@ -806,7 +818,7 @@ int devtelemetry_init(int level)
 {
     if(level == INIT_LEVEL_DEVTELEMETRY)
     {
-#ifndef FEATURE_LAN_ONLY
+#if !defined(FEATURE_LAN_ONLY) && !defined(FEATURE_GUEST_NAT)
```

Comment:
Please update the comment after the matching #endif

Reply:
Also, the comment below has to be updated to explain why the guest_nat feature affects the behavior here